packages/x-date-pickers/src/PickersLayout/PickersLayout.tsx

```diff
@@ -58,8 +59,6 @@ export const PickersLayoutContentWrapper = styled('div', {
   slot: 'ContentWrapper',
   overridesResolver: (props, styles) => styles.contentWrapper,
 })({
-  gridColumn: 2,
```

Comment:
@alexfauquette was there a reason for the coordinates to be defined here but defined on the container for the action bar ? Is this because the file is defined in the same file ?

And are you just `2 2` was the right coordinates ? Shouldn't it be on multiple columns ?

Reply:
As we discussed in the meeting:

- (2,2) is the coordinate for when you need to place components on basic position top, top-right, right, ...
- Yes, the position can be defined somewhere else. I defined it here because it's a styled component so I had the opportunity to add CSS for it. Others are not accessible in this fils, so I used classes